### DISCUSSION TITLE
Question on the usage of --sc

### DISCUSSION THREAD

**Discussion by Chongli-90** [ablab/spades#994] [Q&A] [answered]
Hello!

I have a question on the usage of flag --sc. When I assemble my influenza Illumina 2x 150bp reads, I found it assembled better when I use the flag --sc, especially on avoiding assembling two similar sequencing together (ie, piece of A sequence + piece of B sequence), even though my data is not MDA or single-cell data. Can I use this flag anyway? What changes when I use this flag on the genome assembly? Thank you so much!

Regards,

**Comment by asl** [comment, marked as answer]
Hello

>  I found it assembled better when I use the flag --sc

"Better" as compared to which mode?

**Reply by Chongli-90** [reply]
Hello!

It's better to compare with the default parameters (without add --sc). Thank you so much!

Regards,

**Reply by asl** [reply]
Well, this is pretty much expected :) As default parameters (expect even genome coverage and DNA data which is certainly not true with your data.

For RNA viral assembly we'd suggest RNA viral assembly mode with (in your case) Influenza HMMs as described in  https://pubmed.ncbi.nlm.nih.gov/34406356/

**Reply by Chongli-90** [reply]
Thank you so much for the suggestion! I will look into it. Besides, can I use --sc  flag even if my data is not MDA data? Thanks!

Regards,

**Reply by asl** [reply]
Sure you can (and you already used). But it is not recommended unless you're 100% sure what you're doing

**Reply by Chongli-90** [reply]
I really appreciate your help! Just the final confirmation, compare it with default setting, add --sc will better address the uneven genome coverage data for the samples. Is that correct? Thanks!

**Reply by Chongli-90** [reply]
Also, I use the spades in shovill which will downsampling the genome coverage to 150x and perform the assembly, will that consider as the even coverage? Thank you!

**Comment by asl** [comment]
> I really appreciate your help! Just the final confirmation, compare it with default setting, add --sc will better address the uneven genome coverage data for the samples. Is that correct? Thanks!

We strongly suggest for RNA viral data to use dedicated mode. And neither `--sc` nor default mode.

> I use the spades in shovill which will downsampling the genome coverage to 150x and perform the assembly, will that consider as the even coverage? 

Absolutely not. Also, "downsampling" in shovill will likely lead to suboptimal results in case of uneven coverage. And even coverage gaps.
